components/availability/src/Availability.svelte

```diff
@@ -1222,6 +1221,14 @@
                   class:pending={event_to_hover && slot.selectionPending}
                   data-start-time={new Date(slot.start_time).toLocaleString()}
                   data-end-time={new Date(slot.end_time).toLocaleString()}
+                  class:outside-of-time-range={!slot.fallsWithinAllowedTimeRange}
+                  title={slot.fallsWithinAllowedTimeRange
+                    ? null
```

Comment:
While we're at it, lets add a descriptive title for available slots as well.

cc: @heathergh for accessibility